tartiflette_aiohttp/__init__.py

```diff
@@ -13,6 +30,7 @@ def register_graphql_handlers(
     executor_http_endpoint: str = "/graphql",
     executor_http_methods: List[str] = None,
     engine: Engine = None,
+    graphiql_endpoint: Optional[str] = None,
```

Comment:
What do you think of providing 2 parameters?

* **graphiql_enabled: bool = True**
* **graphiql_options: dict**

**Graphiql options**
* **endpoint_url: str = "/graphiql"**
* **query: str** optional query to pre-populate the GraphiQL UI with
* **operation_name: str** optional operationName to pre-populate the GraphiQL UI with
* **variables: dict** optional variables to pre-populate the GraphiQL UI with
* **pass_headers: List[str]** a string that will be added to the outgoing request header object (e.g "'Authorization': 'Bearer lorem ipsum'")

Inspired by https://www.apollographql.com/docs/apollo-server/v1/graphiql.html

Reply:
I don't really see the point of having "enabled" + "options" if you provide options, it means that you want it. 
If it's disabled by default, providing a graphiql_options != None implicitly enables the thing?
What do you think @tsunammis ?

Reply:
@abusi Which value do you have to provide to enable `graphiql` without specifying any options?

I want to solve 2 things:
- Provide a simple way to enable graphiql without specifying any parameters/options
- Provide an advanced way to configure graphiql with some parameters/options

What do you think of this proposal?

## Graphiql Desactivated
```python
graphql_endpoint: False|None
```

## Simple use-case (graphiql activated)
```python
graphql_endpoint: True
```

## Advanced use-case
```python
graphql_endpoint: {
    "endpoint_url": "/graphiql",
    "query": "blabla",
    "operation_name": "blabla",
    "variables": {},
    "pass_headers": {}
}
```

Reply:
I think you meant `graphiql_endpoint` :).
Thinking about it, let me go back on my first remark, 
providing _enable and _options prevent us for having a parameter that can be of multiple type, the interface is then clearer. 

BUT how do we address the size of the argument list that will continue to grow if we adopt this style of APIS ? 

We'll we go in the directions of a **kwargs at the end and poping intersting keys OR providing an global "options" dict (that we can specify) and inside which we'll have a key for graphiql options.

IE:
```
def ..... options = None):
if not options:
options = {}

graphiqloptions = options.get("graphiql")
if not graphiqloptions:
   graphiqloptions = {"endpoint_url": "/graphiql", "enabled": true} # enabled by default.

if graphiql.get("enabled"):
   manage_graphiql(graphiqloptions)

....

```